### PR TITLE
Update Bitcoin node and Semantra

### PR DESCRIPTION
**Pull request by Dimokus88**
- deleted `sleep infinity`

### CODE REVIEW

File: semantra/deploy.yaml [hunk 1]
```diff
@@ -1,42 +1,33 @@
...
       - "bash"
       - "-c"
     args:
       - 'apt-get update ;
         apt-get upgrade -y ;
-        apt-get install python3 -y;
-        apt-get install pip -y;
-        apt-get install python3.10-venv -y ;
-        apt-get install wget unzip -y ;
+        apt-get install python3  pip python3.10-venv wget unzip -y ;
```
[the remark this answers is not in the document]
Comment (Dimokus88) [reply semantra/deploy.yaml]: Everything is fine with this, added.
I repeat, this is not my template.

File: bitcoin/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+LABEL site="https://declab.pro"
+# Set timezone
+ENV TZ=Europe/London 
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Install necessary packages
+RUN apt update && apt upgrade -y &&\
+    apt install -y wget tar lz4 &&\
+	rm -rf /var/lib/apt/lists/*
+COPY ./init.sh /init.sh
+RUN sed -i 's/\r//' /init.sh && chmod +x /init.sh
```
[the remark this answers is not in the document]
Comment (Dimokus88) [reply bitcoin/Dockerfile]: Accepted and corrected

File: semantra/deploy.yaml [hunk 3]
```diff
@@ -1,42 +1,33 @@
 ---
 version: "2.0"
-
 services:
   app:
-    image: ubuntu:latest     
-    
+    image: ubuntu:22.04
```
[the remark this answers is not in the document]
Comment (andy108369) [reply semantra/deploy.yaml:5]: @Dimokus88 the fix is simple, replace `python3.10-venv` with `python3-venv` ;-)

File: semantra/deploy.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (andy108369) [reply semantra/deploy.yaml]: replace `python3.10-venv` with `python3-venv` so you can bump the ubuntu release from 22.04 to 24.04 etc

**Pull request merged**